Makefile

```diff
@@ -31,7 +31,7 @@ lint: pylint pycodestyle flake8 mypy pytype
 pylint:
 	pylint \
 		--load-plugins pylint_quotes \
-		--disable=W0511,R0801 \
+		--disable=W0511,R0801,cyclic-import \
```

Comment:
With `TYPE_CHECKING`, pylint, different from pytype, also throw a `cyclic-import` error at test stage. When we run plugin program, there is no `cyclic-import` error. So,  I decide to remove pylint `cyclic-import` lint item.